apps/judicial-system/backend/src/app/modules/notification/services/defendantNotification/defendantNotification.service.ts

```diff
@@ -62,14 +62,14 @@ export class DefendantNotificationService extends BaseNotificationService {
     for (const recipient of to) {
       if (recipient.email && recipient.name) {
         promises.push(
-          this.sendEmail(
+          this.sendEmail({
             subject,
-            body,
-            recipient.name,
-            recipient.email,
-            undefined,
-            true,
-          ),
+            html: body,
+            recipientName: recipient.name,
+            recipientEmail: recipient.email,
+            attachments: undefined,
```

Comment:
Consider removing undefined properties as they are no longer needed when passing in an object as a parameter.